app/Http/Controllers/CostumerController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Costumer;
+use Illuminate\Http\Request;
+
+class CostumerController extends Controller
```

Comment:
jgn lupa diganti juga nama filenya jadi CustomerController.php
```suggestion
class CustomerController extends Controller
```